languages/common-lisp/exercises/concept/sameness/sameness-test.lisp

```diff
@@ -0,0 +1,167 @@
+;; Ensures that sameness.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (load "sameness")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from sameness and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage sameness-test
+  (:use :cl :fiveam :sameness)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :sameness-test)
+
+;; forward declaration of test utilities
+(declaim (ftype (function) open-room))
+(defparameter +rooms+ nil)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite sameness-suite)
+(in-suite sameness-suite)
+
+(test the-maze-of-object-identity
```

Comment:
I like this maze structure lots more than the numbered one before! Very nicely done!

Reply:
Yes, once i switched to "rooms" and then switched the tests to "mazes" I really began to like it. The numbering was just no good - but I don't think I could have gotten here without going through the numbering version.